x-pack/plugins/maps/public/classes/layers/solution_layers/security/security_index_pattern_utils.ts

```diff
@@ -29,13 +26,13 @@ export async function getSecurityIndexPatterns(): Promise<IndexPatternMeta[]> {
 
   const indexPatternCache = await getIndexPatternService().getCache();
   return indexPatternCache!
-    .filter((savedObject: SimpleSavedObject<IndexPatternSavedObjectAttrs>) => {
```

Comment:
now uses `SavedObject` instead of `SimpleSavedObject` - type info is available without specifying.